std/container/binaryheap.d

```diff
@@ -265,9 +265,20 @@ and $(D length == capacity), throws an exception.
         }
         else
         {
-            // can't grow
```

Comment:
I didn't know the reason for setting the element after an exception was thrown, so I didn't change the code, but thanks for the heads-up -> changed :)